### PR TITLE
Fixes and includes cleaning.

### PR DESCRIPTION
**Pull request by TonyTEN1**
It includes david's and Sezz's commits.

### CODE REVIEW

**Review by Kubsy — APPROVED** [review]

File: TombEngine/Code/Game/collision/collide_room.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include "Math/Math.h"
+#include <Math/Math.h>
```
Comment (Lwmte) [inline TombEngine/Code/Game/collision/collide_room.h:2]: Please revert.

File: TombEngine/Code/Game/collision/collide_room.cpp [hunk 2]
```diff
@@ -8,7 +7,7 @@
 #include "Game/Lara/lara.h"
 #include "Game/items.h"
 #include "Game/room.h"
-#include "Math/Math.h"
+#include <Math/Math.h>
```
Comment (Lwmte) [inline TombEngine/Code/Game/collision/collide_room.cpp:10]: Please revert.

File: TombEngine/Code/Game/collision/floordata.h [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include "Math/Math.h"
+#include <Math/Math.h>
```
Comment (Lwmte) [inline TombEngine/Code/Game/collision/floordata.h:2]: Please revert.

File: TombEngine/Code/Game/collision/sphere.h [hunk 4]
```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <Math/Math.h>
```
Comment (Lwmte) [inline TombEngine/Code/Game/collision/sphere.h:2]: Please revert.

File: TombEngine/Code/Game/effects/weather.h [hunk 5]
```diff
@@ -1,6 +1,6 @@
 #pragma once
-#include <SimpleMath.h>
-#include "ScriptInterfaceLevel.h"
+
+#include <Scripting/Include/ScriptInterfaceLevel.h>
```
Comment (Lwmte) [inline TombEngine/Code/Game/effects/weather.h:3]: Please revert.

File: TombEngine/Code/Objects/TR5/Emitter/tr5_rats_emitter.h [hunk 6]
```diff
@@ -1,5 +1,6 @@
 #pragma once
-#include "Game/items.h"
+
+#include <Game/items.h>
```
Comment (Lwmte) [inline TombEngine/Code/Objects/TR5/Emitter/tr5_rats_emitter.h:3]: Please revert.

File: TombEngine/Code/Objects/TR5/Emitter/tr5_bats_emitter.h [hunk 7]
```diff
@@ -1,5 +1,6 @@
 #pragma once
-#include "Game/items.h"
+
+#include <Game/items.h>
```
Comment (Lwmte) [inline TombEngine/Code/Objects/TR5/Emitter/tr5_bats_emitter.h:3]: Please revert.

File: TombEngine/Code/Objects/TR5/Emitter/tr5_spider_emitter.h [hunk 8]
```diff
@@ -1,5 +1,6 @@
 #pragma once
-#include "Game/items.h"
+
+#include <Game/items.h>
```
Comment (Lwmte) [inline TombEngine/Code/Objects/TR5/Emitter/tr5_spider_emitter.h:3]: Please revert.

**Review by Lwmte — DISMISSED** [review]
I am against changing directory structure and moving all source files to `Code` subdirectory. `TombEngine` already had source files separated from binary assets or documentation, adding extra directory depth does not clarify anything but introduces more confusion.

**Pull request closed without merging**